Research/Research/RSDOverviewStepObject.swift

```diff
@@ -39,12 +39,18 @@ import Foundation
 open class RSDOverviewStepObject : RSDUIStepObject, RSDOverviewStep {
 
     private enum CodingKeys: String, CodingKey, CaseIterable {
-        case icons
+        case icons, videoResourceName, videoBundleIdentifier
     }
     
     /// The icons that are used to define the list of things you will need for an active task.
     open var icons: [RSDIconInfo]?
     
+    /// Bundle identifier for the video file
+    public var videoBundleIdentifier: String?
+    
+    /// Name of the mp4 resource of the video file
+    public var videoResourceName: String?
```

Comment:
No, sorry. 

First, access to the video should be through the `RSDResourceTransformer` protocol that is used to decode the default bundle elsewhere.

Second, this is a UIAction and would be better included with the other UIAction buttons. Either as a standard navigation or as a custom navigation that is only displayed by overview steps. (Ie. might there be a case where the *next* design will include a video action on a different instruction view or as part of an active step?)

Also, isn't this really just a learn more action?

See `RSDWebViewUIAction`, `RSDWebViewUIActionObject`, `RSDWebViewController`, `actionTapped(with actionType: RSDUIActionType) -> Bool` on `RSDStepViewController`, and `decodeUIAction(from decoder:Decoder, with objectType: RSDUIActionObjectType) throws -> RSDUIAction` on `RSDFactory`.

Reply:
Good thoughts, I like the idea of making a RSDVideoUIAction.  I'll start coding that up.

Also FYI, the learn more action on this screen is already being used by the more info button in the top right.

Reply:
thanks